.circleci/config.yml

```diff
@@ -504,29 +504,41 @@ jobs:
             root: /tmp/workspace
             paths:
                 - repo/stats/
+  waiter:
```

Comment:
Really terrible how hard it is in CircleCI to run a command as a "finally" block... 🤦🏽 
Great work with this workaround though!